.github/workflows/check.yml

```diff
@@ -0,0 +1,29 @@
+name: Check
+
+on:
+  - push
+  - pull_request
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        tox-environment:
+          - docs
+          - examples
+          - lint
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Python ${{ matrix.python-version }}
```

Comment:
This does not look to be defined above?

Reply:
This is indeed not complete. I strongly recommend using https://github.com/ymyzk/tox-gh-actions

I let you check it @jdufresne , if required I could make PR for this PR

Reply:
Thanks for catching this. Fixed in the latest revision.

Reply:
@MRigal I took at look at tox-gh-actions and didn't find it to be an improvement, what in particular are you hoping to see?

To me, it spreads the configuration around. Before all the GH configuration is in the `workflows` directory. But with tox-gh-actions, it now also live in `tox.ini` or `setup.cfg`.

Additionally, different actions have different needs. For example, testing requires installing and running a MongoDB server. But running `flake8` does not require this. So I feel there is some advantage to keeping these configurations separate.